src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -133,14 +133,4 @@ static void DeleteOtherPlatformHost(Permutation permutation)
             }
         }
     }
-
-    [SetUpFixture]
-    public class TestSessionInitializer
-    {
-        [SetUp]
```

Comment:
With V3 I can run this without any errors or warnings:

Note the change from `[Setup]` to `[OneTimeSetUp]`

```CSharp
[SetUpFixture]
public class SetupFixtu
{
    [OneTimeSetUp]
    public void setup()
    {
        
    }
}
```

But like I said, we're only doing a simple initialization of a single variable. Not a major issue either way :smile: